recipe/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "authzed" %}
+{% set version = "1.20.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7610e6b897d4a61765739183e68ca3de32cbc6dc0dc1df247444ae129e58627d
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vvv
+  skip: True  # [py<38]
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+    - poetry-core >=1.0.0
+  run:
+    - python
+    - grpcio >=1.63
+    - protobuf >=5.26,<6
+    - grpc-interceptor >=0.15.4
+    - googleapis-common-protos >=1.65.0
```

```suggestion
    - googleapis-common-protos >=1.65.0
    - protovalidate >=0.7.1
```
Isn't `protovalidate` required too?
https://github.com/authzed/authzed-py/blob/71c97ae94d86aa8e7f54528120a27e61b8b9958a/pyproject.toml#L26

---

Sorry, looks like linked version of dev_url in PR above points to v1.21.1, rather than current version 1.20.0.
This dep requirement was only introduced after current version of 1.20.0

---

* updated the links above to point to correct version